src/main/java/TurnUseCases/MortgageUseCase/MortgageProperty.java

```diff
@@ -21,6 +21,19 @@ public void mortgage(Player player, Property property){
         }
     }
 
+
+    /**
+     * Overloading method mortgage. Player mortgages a color property.
+     *
+     * @param player the player who wants mortgages property.
+     * @param property the property which player wants to mortgages.
+     */
+    public void mortgage(Player player, ColorPropertyTile property){
+        if (player.ownsProperty(property) && property.getHouse() == 0 && property.getHotel() == 0) {
+            player.addMoney(property.mortgage());
+        }
+    }
+
```

Comment:
Since ColorPropertyTile inherits from Property when the program compiles, overloading the mortgage method may cause some compilation issues. It would be better to directly search for a colored property.